### DISCUSSION TITLE
What is main difference with going serverless with Vercel?

### DISCUSSION THREAD

**Discussion by jysung** [serverless-nextjs/serverless-next.js#2474] [Q&A]
I know the pricing structure with Vercel is different and I would get more control/insights into each lambdas with serverless-nextjs, is there any major benefits or difference of `serverless-next.js` and Vercel?

**Comment by khuezy** [comment]
Vercel will always have the latest features while serverless-next (and others) will always be behind, especially major features like middleware.
